routes/api.php

```diff
@@ -11,25 +11,49 @@
 |
 */
 
+use App\Http\Controllers\API\AuthController;
 use App\Http\Controllers\API\v1\EventController;
+use App\Http\Controllers\API\v1\LikesController;
 use App\Http\Controllers\API\v1\StatisticsController;
 use App\Http\Controllers\API\v1\StatusController;
 use App\Http\Controllers\API\v1\UserController;
+use App\Http\Controllers\API\v1\AuthController as v1Auth;
 use Illuminate\Support\Facades\Route;
 
 Route::group(['prefix' => 'v1', 'middleware' => 'return-json'], function() {
-    Route::get('statuses', [StatusController::class, 'enRoute']);
-    Route::get('statuses/{id}', [StatusController::class, 'show']);
-    Route::get('stopovers/{parameters}', [StatusController::class, 'getStopovers']);
-    Route::get('polyline/{parameters}', [StatusController::class, 'getPolyline']);
-    Route::get('event/{slug}', [EventController::class, 'show']);
-    Route::get('event/{slug}/statuses', [EventController::class, 'statuses']);
-    Route::get('user/{username}', [UserController::class, 'show']);
-    Route::get('user/{username}/statuses', [UserController::class, 'statuses']);
-    Route::get('leaderboard', [StatisticsController::class, 'leaderboard']);
-    Route::get('leaderboard/{month}', [StatisticsController::class, 'leaderboardForMonth']);
-    Route::get('leaderboard/distance', [StatisticsController::class, 'leaderboardByDistance']);
-    //Route::get('leaderboard/friends', [StatisticsController::class, 'leaderboardFriends']); ToDo: Friends route
+    Route::group(['prefix' => 'auth'], function() {
+        Route::post('login', [v1Auth::class, 'login']);
+        Route::post('signup', [v1Auth::class, 'signup']);
+        Route::group(['middleware' => 'auth:api'], function() {
+            Route::post('refresh', [v1Auth::class, 'refresh'])->middleware('auth:api');
+            Route::post('logout', [v1Auth::class, 'logout'])->middleware('auth:api');
+            Route::get('user', [v1Auth::class, 'user'])->middleware('auth:api');
```

Comment:
Already specified in the group.
```suggestion
            Route::post('refresh', [v1Auth::class, 'refresh']);
            Route::post('logout', [v1Auth::class, 'logout']);
            Route::get('user', [v1Auth::class, 'user']);
```